app/questionnaire/relationship_location.py

```diff
@@ -12,13 +12,13 @@ class RelationshipLocation:
     block_id: str
     list_name: str
     list_item_id: str
-    to_list_item_id: Optional[str] = None
+    to_list_item_id: str | None = None
 
-    def for_json(self) -> Mapping:
+    def for_json(self) -> Mapping[str, Any]:
         attributes = vars(self)
         return {k: v for k, v in attributes.items() if v is not None}
 
-    def url(self, **kwargs) -> str:
+    def url(self, **kwargs: Any) -> str:
```

Comment:
it could include _external which is a bool, I wasn't able to type it because if I specify str | None | bool mypy complains not knowing which is which and I can't edit the method itself, open to any other ideas though

Reply:
Ah okay, happy to keep as is then 👍